### PR TITLE
Fix compilation error when having getTemplate() property (issue #48)

### PR DESCRIPTION
**Pull request by dorireuv**
Fix issue #48: Compilation error when having getTemplate() property.

### CODE REVIEW

File: src/test/java/org/inferred/freebuilder/processor/ListMultimapPropertyFactoryTest.java [hunk 1]
```diff
@@ -628,11 +628,11 @@ public void testMergeFrom_builder() {
         .with(new Processor())
         .with(MULTIMAP_PROPERTY)
         .with(testBuilder()
-            .addLine("DataType.Builder template = DataType.builder()")
+            .addLine("DataType.Builder _template = DataType.builder()")
```
Comment (cjp39) [inline src/test/java/org/inferred/freebuilder/processor/ListMultimapPropertyFactoryTest.java]: Could you revert all the changes to tests that are just using a variable named "template"?

**Comment by cjp39** [comment]
Thanks, Dori! It just needs a new test in ProcessorTest.java of a value type with a getTemplate() method, to ensure we don't regress in future.

**Comment by cjp39** [comment]
Also, don't forget to sign the [Google Individual Contributor License Agreement](https://developers.google.com/open-source/cla/individual?csw=1)

**Comment by dorireuv** [comment]
Reverted the unrelated changes and added a test (verified that it fails before the fix).
I also signed the Google Individual Contributor License Agreement.

**Comment by cjp39** [comment]
Perfect; thanks!

**Pull request merged**